assets/js/base/components/cart-checkout/place-order-button/index.tsx

```diff
@@ -1,15 +1,20 @@
 /**
  * External dependencies
  */
+import classnames from 'classnames';
 import { useCheckoutSubmit } from '@woocommerce/base-context/hooks';
 import { Icon, check } from '@wordpress/icons';
 import Button from '@woocommerce/base-components/button';
 
 interface PlaceOrderButton {
 	label: string;
+	fullWidth?: boolean;
```

Comment:
Since this is an optional prop, you need to type it `fullWidth?: boolean | undefined`

This is due to [this tsconfig rule](https://github.com/woocommerce/woocommerce-blocks/blob/6f6c21cf1036ae7b5865ae43406dc0b507f7b5ef/tsconfig.base.json#L10)

```suggestion
	fullWidth?: boolean | undefined;
```

Reply:
Done in https://github.com/woocommerce/woocommerce-blocks/pull/9721/commits/d0848f5ef2fc9a430ab0c7f224eae3c363034e10